src/encoded/schemas/library.json

```diff
@@ -282,6 +282,12 @@
             "type": "string",
             "pattern": "(^[0-9]+-[0-9]+$|^[<>][0-9]+$)"
         },
+        "queried_RNP_size_range": {
+            "title": "Queried RNP size range",
+            "description": "The measured size range of queried ribonucleoprotein (RNP) eCLIP control, in kD.",
+            "type": "string",
+            "pattern": "(^[0-9]+-[0-9]+$|^[<>][0-9]+$)"
```

Comment:
I think we should go with a restrictive regular expression and adjust it only if there will be need to do that. Please leave only ^[0-9]+-[0-9]+$ and not ^[<>][0-9]+$